src/Test/Perf/ConsumptionParser/ConsumptionParser.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+using System;
+using System.Collections.Generic;
+using System.Xml.Linq;
+
+namespace Roslyn.Test.Performance.Utilities.ConsumptionParser
+{
+
+    internal struct ConsumptionParse
+    {
+        public IEnumerable<ScenarioResult> Scenarios { get; }
+        public ConsumptionParse(IEnumerable<ScenarioResult> scenarios)
```

Comment:
My internal use of this constructor passes in a Linq type.  Not sure if I want that as the parameter type?